src/Java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/nbthandlers/GT_MetaTileEntity_Hatch_NbtConsumable.java

```diff
@@ -40,13 +40,6 @@ public GT_MetaTileEntity_Hatch_NbtConsumable(String aName, int aTier, int aInput
 		mAllowDuplicateUsageTypes = aAllowDuplicateTypes;
 	}
 
-	public GT_MetaTileEntity_Hatch_NbtConsumable(String aName, int aTier, int aInputSlots, String[] aDescription, boolean aAllowDuplicateTypes, ITexture[][][] aTextures) {
```

Comment:
this is literally duplicate of the function above

Reply:
It’s a constructor. They wouldn’t be identical.

Reply:
constructor good 
intelij bad